From f95925afef451755fc1449e57fbfdc2e7277f4b4 Mon Sep 17 00:00:00 2001
From: Maxim Cournoyer <maxim.cournoyer@gmail.com>
Date: Sat, 11 Jan 2020 00:44:55 -0500
Subject: [PATCH 1/3] tests: Link against GLib.

This resolves an error about not finding <glib.h> when linking the
tests that make use of GLib.

* src/tests/CMakeLists.txt[2GEOM_GTESTS_SRC]: Add the glib library to
the link target libraries.
---
 src/tests/CMakeLists.txt | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/src/tests/CMakeLists.txt b/src/tests/CMakeLists.txt
index 5ddf7f9c..626cfd87 100644
--- a/src/tests/CMakeLists.txt
+++ b/src/tests/CMakeLists.txt
@@ -32,8 +32,10 @@ sbasis-test
 
 foreach(source ${2GEOM_GTESTS_SRC})
 	add_executable(${source} ${source}.cpp)
-	target_include_directories(${source} PRIVATE ${GSL_INCLUDE_DIRS} ${GTK3_INCLUDE_DIRS})
-	target_link_libraries(${source} 2geom gtest_main gtest ${GSL_LIBRARIES} ${GTK3_LIBRARIES})
+	target_include_directories(${source} PRIVATE ${GSL_INCLUDE_DIRS}
+	  ${GTK3_INCLUDE_DIRS} ${GLIB_INCLUDE_DIRS})
+	target_link_libraries(${source} 2geom gtest_main gtest ${GSL_LIBRARIES}
+	  ${GTK3_LIBRARIES} ${GLIB_LIBRARIES})
 	add_test(NAME ${source} COMMAND ${source})
 endforeach()
 
-- 
2.24.1

